service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,140 @@
+
+    <service verb="get" noun="LatestOrders">
+        <out-parameters>
+            <parameter name="ordersList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <order-by field-name=" -placedDate"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- 4.Finding the Order Header details for a given orderId.-->
+
+    <service verb="get" noun="Order">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="ordersList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList"/>
+            <econdition field-name="orderId"/>
+        </actions>
+    </service>
+
+    <!-- 5.Finding all Order Header records with the grandTotal greater than 50.-->
+
+    <service verb="get" noun="OrderHeaders">
+        <out-parameters>
+            <parameter name="orderHeaderDetails" type="list"/>
+        </out-parameters>
+        <actions>
+
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <econdition field-name="grandTotal" operaor="greater" from="50"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- @  6.Listing all Order Item details for the order with orderId, 100102 and orderPartSeqId, 01.-->
+    <service verb="get" noun="OrderItems">
+        <out-parameters>
+            <parameter name="orderItemList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderItem" list="orderItemList">
+                <econdition field-name="orderId" value="100102"/>
+                <econdition field-name="orderPartSeqId" value="01"/>
+            </entity-find>
+        </actions>
+    </service>
+
+  <!--7 Find all Order Parts with partTotal less than equals to 20.-->
+
+    <service verb="get" noun="OrderParts">
+        <out-parameters>
+            <parameter name="orderPartDetails" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="orderPartDetails">
+                <econdition field-name="orderId" operaor="less" from="20"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- @ 8.Find all Order Part records assigned to the facility with ZIRET_WH value-->
+
+    <service verb="get" noun="OrderPartData">
+        <in-parameters>
+            <parameter name="facilityId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="OrderPartData" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="OrderPartData">
+            <econdition field-name="facilityId" value="ZIRET_WH"/>
```

Comment:
Can you check if how can we return only selected fields from OrderPart as was expected for this query?

Reply:
we can use select field here,completed